src/plugins/analysis/qemu_exec/docker/start_binary.py

```diff
@@ -1,17 +1,18 @@
+from __future__ import annotations
```

Comment:
Flake8 complains quite rightly that the shebang should be at the top of the file